san/sanbase_graphql_helper.py

```diff
@@ -0,0 +1,268 @@
+import iso8601
+import datetime
+
+_DEFAULT_INTERVAL = '1d'
```

Comment:
You no longer need to prepend `_` to functions and vars in this module

Reply:
I do, because when I import in sanbase_graphql from sanbase_graphql_helper, DEFAULT_INTERVAL gets imported needlessly, hence the '_'.

Reply:
if it starts with `_` it's not imported?

Reply:
Yes, exactly

Reply:
Then why you needed to remove the functions starting with `_` in the previous implementation?

Reply:
Because inspect.getmember returns all of the functions (there isn't any real encapsulation in python), but when you import stuff from another module, python is made so, that you cannot import functions starting with '_' or '__'